specs/agents/mobile/README.md

```diff
@@ -20,24 +20,26 @@ However, the Elastic mobile agents don't set every attribute defined (many only
+| [`os.version`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)           | `host.os.version`                 | `15.5.0`                               | :x: no | |
+| [`os.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/os/)              | `host.os.name` and `host.os.type` | `iOS`, `Android`                       | :white_check_mark: yes |  |
+| [`deployment.environment`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/deployment_environment/) | `service.environment`             | `production`, `dev`                    | :x: no |    |
+| [`device.id`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)        | `device.id`                       | `E733F41E-DF47-4BB4-AAF0-FD784FD95653` | :x: no |  [Follow this description.](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/) (Device ID should be fix and unique for a device but should not carry PII) |
+| [`device.model.identifier`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/) | `device.model.identifier`         | `iPhone4`,`SM-G920F`                   | :x: no  |             |
+| [`device.model.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)       | `device.model.name`               | `Samsung Galaxy S6`                    | :x: no  | This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |
+| [`device.manufacturer`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/device/)     | `device.manufacturer`             | `Apple`, `Samsung`                     | :x: no  |  This information is potentially not directly available on the device and needs to be derived / mapped from `device.model.identifier`. In this case, the APM server should do the mapping. |   
+| [`process.runtime.name`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/process/#process-runtimes) | `service.runtime.name`            | `Android Runtime`                      | :x: no | Use `Android Runtime` for Android. For iOS use `iOS`. |
+| [`process.runtime.version`](https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/process/#process-runtimes) | `service.runtime.version`         | `2.0.1`                                | :x: no | Use the Dalvik version for Android (`System.getProperty("java.vm.version")`). For iOS use the version of iOS. |
+| `session.id`                             | `session.id`                       | `A73DC41E-DF18-4BB4-ABC0-F0000FD95653` | :x: no                            |                                                                                                               | 
```

Comment:
Is the iOS agent setting `session.id` in the OTel resources? Because at the moment the Android one is just adding it as a regular, dynamic attr.

Reply:
No, we aren't, but for lack of a better place for this attribute, I threw it in this table. It's probably more appropriate from a categorization perspective to put it in it's own table, but I'm not sure if there's a term in open-telemetry for a non-resource attribute that applies to all signals.

Reply:
Oh! Got it, makes sense. I just wanted to double-check because it was a bit confusing to see it under the resource attrs, though I see what you mean, it's not clear what table it should go to.